sync/downstream.py

```diff
@@ -945,6 +946,10 @@ def commit_status_changed(git_gecko, git_wpt, sync, context, status, url, head_s
             sync.update_commits()
             sync.update_github_check()
             return
+
+        if context == "Taskcluster (pull_request)" and status == "success":
+            get_temporary_metadata(sync)
```

Comment:
We probably want to ensure we only do this once per push per sync.

Reply:
Doesn't this already check for this, as it is checking if the taskcluster run has completed and then attempts to get the metadata.